Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY --from=certificates /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-ce
 
 # Copy binary built on the host
 COPY bin/manager_${TARGETARCH} manager
+COPY bin/must-gather_${TARGETARCH} must-gather
```

Comment:
Does this need to be a part of the operator image? I'd prefer not to make this image larger for all users.

Reply:
It is a good practice and can help with debuggability. But I'm ok with removing it or adding something to include the binary in a conditional way.

Reply:
How big is it?

Reply:
```
.rwxr-xr-x@ 39M iblancasa 27 Aug 15:54 bin/must-gather_amd64
```

Reply:
I can add a new container or, maybe, add something to add the binary only if you want. Like a `makefile` variable or something like that.

Reply:
yeah i think if we can indeed build this as a separate image that would be better. Especially given not all users will want/need this functionality. We also can modify the openshift bundle to automatically use this if we'd like too.